configuration/rules-dsl.md

```diff
@@ -206,6 +206,40 @@ For example, one z-wave device can be "zwave:device:c5155aa4:node14".
 > Note: You need to use quotes around `thingUID` if it contains special characters such as ':'.
 > The parser will not recognize the whole text correctly without quotes.
 
+{: #channel-based-triggers}
+### Channel-based Triggers
+
+You can listen for triggers from specific *trigger channels* that are provided by some addons.
```

Comment:
Consider replacing with the following:
"Some addons (add-ons?) provide *trigger channels*.  Your rules can take actions based upon trigger events generated by these trigger channels."

Reply:
👍 I also prefer the "Your rule can" vs "You can". Feel free to also improve the Thing-based Triggers